Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeInformation.ps1

```diff
@@ -196,73 +113,82 @@ function Get-ExchangeInformation {
+                $extendedProtectionConfig = Get-ExtendedProtectionConfiguration @getExtendedProtectionConfigurationParams
             } catch {
                 Write-Verbose "Failed to get the ExtendedProtectionConfig"
                 Invoke-CatchActions
             }
         }
 
-        $exchangeInformation.ApplicationConfigFileStatus = Get-ExchangeApplicationConfigurationFileValidation -ComputerName $Server -ConfigFileLocation ("{0}EdgeTransport.exe.config" -f $serverExchangeBinDirectory)
-
-        $buildInformation.KBsInstalled = Get-ExchangeUpdates -Server $Server -ExchangeMajorVersion $buildInformation.MajorVersion
-        if (($null -ne $buildInformation.KBsInstalled) -and ($buildInformation.KBsInstalled -like "*KB5000871*")) {
-            Write-Verbose "March 2021 SU: KB5000871 was detected on the system"
-            $buildInformation.March2021SUInstalled = $true
-        } else {
-            Write-Verbose "March 2021 SU: KB5000871 was not detected on the system"
-            $buildInformation.March2021SUInstalled = $false
-        }
-
-        Write-Verbose "Checking if FIP-FS is affected by the pattern issue"
-        $fipFsParams = @{
-            ComputerName   = $Server
-            ExSetupVersion = $buildInformation.ExchangeSetup.FileVersion
-            ServerRole     = $buildInformation.ServerRole
-        }
-
-        $buildInformation.FIPFSUpdateIssue = Get-FIPFSScanEngineVersionState @fipFsParams
+        $applicationConfigFileStatus = Get-ExchangeApplicationConfigurationFileValidation -ComputerName $Server -ConfigFileLocation ("{0}EdgeTransport.exe.config" -f $serverExchangeBinDirectory)
+        $serverMaintenance = Get-ExchangeServerMaintenanceState -Server $Server -ComponentsToSkip "ForwardSyncDaemon", "ProvisioningRps"
+        $settingOverrides = Get-ExchangeSettingOverride -Server $Server -CatchActionFunction ${Function:Invoke-CatchActions}
 
-        if (($buildInformation.MajorVersion -ge [HealthChecker.ExchangeMajorVersion]::Exchange2016) -and
-        ($buildInformation.ServerRole -ne [HealthChecker.ExchangeServerRole]::Edge)) {
+        if (($versionInformation.BuildVersion -ge "15.1.0.0") -and
+        ($getExchangeServer.IsEdgeServer -eq $false)) {
             Write-Verbose "SerializedDataSigning must be configured via SettingOverride"
-            $exchangeInformation.SerializationDataSigningConfiguration = Get-ExchangeSerializedDataSigningState -GetSettingOverride $PassedOrganizationInformation.SettingOverride
-        } elseif (($buildInformation.MajorVersion -eq [HealthChecker.ExchangeMajorVersion]::Exchange2013) -and
-            ($buildInformation.ServerRole -ne [HealthChecker.ExchangeServerRole]::Edge)) {
+            $serializationDataSigningConfiguration = Get-ExchangeSerializedDataSigningState -GetSettingOverride $PassedOrganizationInformation.SettingOverride
```

Comment:
```suggestion
            [array]$serializationDataSigningConfiguration = Get-ExchangeSerializedDataSigningState -GetSettingOverride $PassedOrganizationInformation.SettingOverride
```

Make this an array as it was in class. Otherwise, the check in `Invoke-AnalyzerSecuritySerializedDataSigningState` would fail as we don't have the `Count()` method available:
![image](https://user-images.githubusercontent.com/40993616/217827461-61f77778-836b-494d-b387-1b3045137d74.png)